pwa-devdocs/src/tutorials/pwa-studio-fundamentals/add-a-static-route/index.md

```diff
@@ -0,0 +1,284 @@
+
+import React, { useEffect } from 'react';
+
+const Foo = () => {
+
+  useEffect(() => {
+    document.title = 'Foo Test Page';
+  }, []);
+
+  return (
+    <h1>Hello World JSX</h1>
+  );
+}
+
+export default Foo;
+```
+
+### Create `index.js` file
+
+Inside the Foo component directory, create an `index.js` file that exports the Foo component.
+This pattern of exposing the module through the `index.js` file is the same pattern used in the Venia UI package.
+
+```js
+/* src/components/Foo/index.js */
+
+export {default} from './foo';
+```
+
+## Overriding Venia components overview
+
+As described in the [Project Structure][] topic, a new project set up using the scaffolding command imports the entire Venia app as a single component.
+To replace the [Routes][] component, which is imported and used in Venia's [App][] component, you need to:
+
+1.  Define a new Route component
+2.  Make a copy of the App component that imports your new Route component
+3.  Import your project's App component in yougr `src/index.js` file
+
+Copies of the Route and App components are found in your project's `node_modules` directory.
+
+## Override Routes component
+
+The Routes component is responsible for deciding which components to use for rendering non-Magento routes.
+To override this component, you need to copy the source code into your project and make the necessary modifications.
+
+### Create Routes directory
+
+Create a Routes directory under components. This will hold your project's version of the Routes component.
+
+```sh
+mkdir -p src/components/Routes
+```
+
+### Copy over Routes component code
+
+Get a copy of the `routes.js` file from the `node_modules` directory:
+
+```sh
+cp node_modules/@magento/venia-ui/lib/components/Routes/routes.js src/components/Routes
+```
+
+### Update Routes's module imports
+
+Open the `routes.js` file and update the relative import statements to use components from the Venia UI package.
+
+```diff
+- import { fullPageLoadingIndicator } from '../LoadingIndicator';
+- import MagentoRoute from '../MagentoRoute';
++ import { fullPageLoadingIndicator } from '@magento/venia-ui/lib/components/LoadingIndicator';
++ import MagentoRoute from '@magento/venia-ui/lib/components/MagentoRoute';
+ 
+- const CartPage = lazy(() => import('../CartPage'));
+- const CheckoutPage = lazy(() => import('../CheckoutPage'));
+- const CreateAccountPage = lazy(() => import('../CreateAccountPage'));
+- const Search = lazy(() => import('../../RootComponents/Search'));
++ const CartPage = lazy(() => import('../CartPage'));
++ const CheckoutPage = lazy(() => import('../CheckoutPage'));
++ const CreateAccountPage = lazy(() => import('../CreateAccountPage'));
++ const Search = lazy(() => import('../../RootComponents/Search'));
+```
+
+### Import your custom component
+
+Import your custom Foo component in the `routes.js` file.
+Use lazy loading to avoid unneccesary loading of your component whenever the Routes component is imported.
```

Comment:
```suggestion
Use lazy loading to load your component only when it is needed.
```

While the existing copy is technically true, I think this is more straightforward and impactful.